### PR TITLE
Led light brightness nerf + power increase

### PR DESCRIPTION
**Pull request by Peptide90**
## About the PR 
Self explanatory, I wanted LED lights originally for a brighter, more space ship looking station. Fairly certain when the PR went up we agreed to reduce the brightness though as 0.8 is our current lights and the LED light is 4. (#5869)

First screenshot below shows the lights before this change (very washed out and white) and after in the second as I reduced it to 2.5, so they're still very good light at over 3x brighter than standard so good for hallways and work areas like surgery.

Also took the liberty to increase all lights to 60 watts from 40 so power use across the station is slightly higher (we need more things to draw power) and changed LED lights from 9 watts to 80 (Yes conventionally LED lights are low power for the same brightness but these are industrial lights and should have a negative to them since you need less lights overall anyway). ( #5870)
**Screenshots**

**BEFORE**
power 4
**AT 2.5**

A new exterior light has been added with a blue tint so light from outside is distinct (for shuttles etc).
#9cbcff
Went the whole hog and added sodium lamps too for that horrible machinery room or scrap shuttle look.
sodium

**Changelog**

:cl:
- fix: Lights now consume slighly more power than before. Can you cope?
- fix: LED lights are now less bright so won't burn your eyes out, just your hands.
- add: There's two high energy lightbulb specifically for shuttles and station exteriors. Spot the blue and orange glow!

### CODE REVIEW

**Review by Pangogie — COMMENTED** [review]
Why do LED bulbs use more power than normal bulbs?

**Comment by Peptide90** [comment]
Gameplay really. Having a 3x bulb that uses next to no power seems bad and industrial LEDs are still high wattage.

I could drop it down to like 30W so it's more realistic but imo you'll need half as many normal lights by switching to LED so you're still net positive on power use.

**Comment by Peptide90** [comment]
Changed to DNM for now as I'm going to also add a high power high range wall mount floodlight for use on ships and station exterior etc.

Might also revising LED power draw.

**Comment by mirrorcult** [comment]
should ask around in discord if anyone has sprites for the exterior lights/sodium lamps

**Comment by Peptide90** [comment]
> should ask around in discord if anyone has sprites for the exterior lights/sodium lamps

I can whip up a square bulkhead type light in a cage if that's preferable for the exterior lights instead of everything being a tube light variant?

File: Resources/Prototypes/Entities/Objects/Power/lights.yml [hunk 1]
```diff
@@ -95,16 +94,53 @@
...
+    
+- type: entity
+  parent: BaseLightbulb
+  name: sodium light tube
+  description: A high power high energy bulb for the depths of space. Salty.
+  id: SodiumLightTube
+  components:
+  - type: LightBulb
+    bulb: Tube
+    color: "#FF7E00"
```
Comment (Pangogie) [inline Resources/Prototypes/Entities/Objects/Power/lights.yml:138]: I would like to propose `#FFAF38` which is[ taken from a blog](http://planetpixelemporium.com/tutorialpages/light.html) and adjusted to be slightly more orange.
image

File: Resources/Prototypes/Entities/Objects/Power/lights.yml [hunk 2]
```diff
@@ -95,16 +94,53 @@
...
+
+- type: entity
+  parent: BaseLightbulb
+  name: exterior light tube
+  description: A high power high energy bulb for the depths of space. May contain mercury.
+  id: ExteriorLightTube
+  components:
+  - type: LightBulb
+    bulb: Tube
+    color: "#9cbcff"
```
Comment (Pangogie) [inline Resources/Prototypes/Entities/Objects/Power/lights.yml:120]: I would also like to propose `#B4FCF0`.
image

Comment (Peptide90) [reply Resources/Prototypes/Entities/Objects/Power/lights.yml]: Done :)

**Pull request merged**